shell/platform/android/android_context_gl.cc

```diff
@@ -105,10 +106,119 @@ static bool TeardownContext(EGLDisplay display, EGLContext context) {
   return true;
 }
 
+class AndroidEGLSurfaceDamage {
```

Comment:
any reason this class is forward declared rather than have the definition in `android_context_gl.h`?

Reply:
I prefer not having to split it into header/implementation. I don't have any strong feeling about it, if you prefer the definition in header I can certainly do it.